Realm/RLMRealm.h

```diff
@@ -337,7 +352,19 @@ typedef void(^RLMNotificationBlock)(NSString *notification, RLMRealm *realm);
 /**
  Helper to perform a block within a transaction.
  */
-- (void)transactionWithBlock:(void(^)(void))block;
+- (void)transactionWithBlock:(void(^)(void))block RLM_SWIFT_UNAVAILABLE("");
```

Comment:
The message will never actually be shown or seen, since `NS_SWIFT_UNAVAILABLE` is only defined starting in Xcode 7, and the non-error APIs will be shadowed by the error ones.